lms/static/js/spec/student_profile/helpers.js

```diff
@@ -105,9 +105,78 @@ define(['underscore'], function(_) {
         expect(learnerProfileView.$('.wrapper-profile-section-two').length).toBe(0);
     };
 
+    var expectModeToggleToBeHidden = function(modeToggleView) {
+        expect(modeToggleView.$el.hasClass('is-hidden')).toBe(true);
+    };
+
+    var expectModeToggleToBeShown = function(modeToggleView) {
+        expect(modeToggleView.$el.hasClass('is-hidden')).toBe(false);
+    };
+
+    var expectBadgesDisplayed = function(badgeListingView, learnerProfileView, empty) {
+        expect(learnerProfileView.$el.find('.wrapper-profile-section-two').is(':visible')).toBe(false);
+        expect(badgeListingView.$el.is(':visible')).toBe(true);
+        if (empty) {
+            expect(badgeListingView.$el.find('.badge-display').length).toBe(1);
+        } else {
+            expect(badgeListingView.$el.find('.badge-display').length).toBe(4);
+        }
+        expect(badgeListingView.$el.find('.find-button-container').length).toBe(1);
+    };
+
+    var expectBadgesHidden = function(badgeListingView, learnerProfileView) {
+        expect(badgeListingView.$el.is(':visible')).toBe(false);
+        expect(learnerProfileView.$el.find('.wrapper-profile-section-two').is(':visible')).toBe(true);
+    };
+
+    var exampleBadges = [{
+        "user": "http://localhost:8000/api/mobile/v0.5/users/honor",
+            "badge_class": {
+                "slug": "test_slug_0_0753941808621",
+                "issuing_component": "test_component",
+                "display_name": "Test Badge",
+                "course_id": null,
+                "description": "Yay! It's a test badge.",
+                "criteria": "https://example.com/syllabus",
+                "image": "http://localhost:8000/media/badge_classes/test_lMB9bRw.png"
```

Comment:
@nasthagiri Looks like my change got clobbered by some git operation. I'll go fix it.